src/python3_pip_skeleton/__main__.py

```diff
@@ -234,6 +242,11 @@ def main(args=None):
     sub.add_argument(
         "--package", default=None, help="Package name, defaults to directory name"
     )
+    sub.add_argument(
+        "--from-branch",
+        default=None,
```

Comment:
Is there a reason you couldn't just write `"main"` here?

Reply:
No, but that would be simpler I agree

Reply:
Oh I remember, it was to keep it consistent with the existing code. Notice how all arguments default to None and get set later in merge_skeleton.